src/mlpack/bindings/R/mlpack/tests/testthat/test-object_type.R

```diff
@@ -0,0 +1,10 @@
+test_that("Test for mlpack S3 class ", {
```

Comment:
```suggestion
test_that("TestS3Class", {
```
Up to you on the pedantry here, but this matches the naming of the other tests.

Reply:
Done in https://github.com/mlpack/mlpack/commit/3acac5b827a2d8289caa8409b18caf9c1130a495